frontend_vue/src/components/tokens/my_sql/howToUse.ts

```diff
@@ -0,0 +1 @@
+export const howToUse = ['Add here'];
```

Comment:
for this stuff i like the separate files. having this separate from the manage / display seems a bit much? lets just include the howToUse stuff in one of the other other components? Is there any reason to have it separate?

Reply:
the ModalContentHowToUse component takes care of styling a holding the list.
The list in the end is just text, so for every token, I dynamically load this file into the modal.